src/data/markdown/docs/03 cloud/02 Analyzing Results/08 Test Comparison.md

```diff
@@ -110,32 +110,48 @@ You can add extra charts, such as timing breakdowns for each HTTP request.
 
 ![k6 Cloud Results: HTTP comparison](./images/07-Test-Comparison/http-comparison.png)
 
-## Compare a series of tests
+## Compare all runs for a test
 
-To compare multiple tests across time, use the *performance-trending chart*.
-The chart shows test times and uses colors to signal the status of a specific test.
+To compare runs for a test across time, use the *performance-trending chart*.
+The chart displayes test run metrics and uses colors to signal the status of a specific run.
 
-To view the performance-trending graph, use the Project page, or select the name of any given test.
-In both cases, k6 plots the `p95` response time metric for all HTTP requests from the test run.
+To view the performance-trending chart for multiple tests belonging to a same project, open the Project page:
 
-### Dashboard performance trending
+![Project Performance Trending](./images/09-Performance-Trending/perf-trending-project.png)
 
-The top-level page for a project shows all project tests and run statuses.
-It provides a high-level view of how performance trends over time.
+Additionally, in order to view the performance-trending chart for an individual test, open the test's page:
 
-In this example, the script `api.js` is running its test, `Demo with Cloud Execution`.
-Though its previous runs had quite a few failures,
-this run is passing and trending down&mdash;a good sign.
+![Test Performance Trending](./images/09-Performance-Trending/perf-trending-test.png)
 
-![Dashboard Performance Trending](./images/09-Performance-Trending/dashboard-perf-trending.png)
+This last version of the chart shows more data points over time. For more information on an individual run, hover over any bar in the chart.
 
-## Test performance trending
+The data displayed in the performance-trending chart is `p95` of the HTTP response time (`http_req_time`) for responses with a status code (so, for example, requests that timed out are not considered).
 
-For a performance-trending graph of a specific test run, select the test.
-This graph shows more data points over time.
+We call this combination of metric plus aggregation function the _trending metric_ of the chart. k6 produces a single value for each test run using the trending metric, and then plots each value the chart.
```

Comment:
I want to maintain the term "trending metric" when possible, so I will write it like this:
summary _trending metric_
wdyt?

Reply:
Also changing "test" for "test run"

Reply:
Whether to italicize the whole phrase or just _trend metric_ is not something I feel strongly about though haha.

Reply:
I agree with your explanation, it's just that the UI uses the term "trending metric" on many different places so I thought it would be best to at least mention the term explicitly once in the docs, to make it feel more connected.

Reply:
Oh, I see I didn't think about the UI text. I think your proposal is reasonable here. I updated my suggestion.